spec/models/manageiq/providers/ansible_tower/automation_manager/refresher_spec.rb

```diff
@@ -116,18 +116,18 @@ def assert_configuration_script_with_survey_spec
     expect(system).to have_attributes(
       :name        => "Ansible-JobTemplate-Survey",
       :description => "Ansible-JobTemplate-Description",
-      :manager_ref => "155",
+      :manager_ref => "81",
       :variables   => {'abc' => 123}
     )
     survey = system.survey_spec
     expect(survey).to be_a Hash
-    expect(survey['spec'].first['index']).to eq 0
```

Comment:
`index` was dropped from the results. 
Before it was used to indicate the order of the returned hash. Now they are just in order.

I'm not sure if that influences our UI part?

Plus I could not figure out if we are creating surveys - that might be a problem if we provide an `index` key